addons/color-picker/paint-editor.js

```diff
@@ -36,7 +36,7 @@ export default async ({ addon, console, msg }) => {
         const previousTool = addon.tab.redux.state.scratchPaint.color.eyeDropper.previousTool;
         if (previousTool) previousTool.activate();
         addon.tab.redux.state.scratchPaint.color.eyeDropper.callback(
-          tinycolor(hex).setAlpha(scratchAddons.opacitySliderAlpha).toRgbString()
+          tinycolor(hex).setAlpha(scratchAddons.opacitySliderAlpha).toHexString()
```

Comment:
`toHexString` ignores the alpha channel, so this will always return the same result as just `tinycolor(hex).toHexString()`.